code/modules/clothing/outfits/event.dm

```diff
@@ -0,0 +1,33 @@
+/datum/outfit/santa //ho ho ho!
+	name = "Santa Claus"
+
+	uniform = /obj/item/clothing/under/color/red
+	shoes = /obj/item/clothing/shoes/sneakers/red
+	suit = /obj/item/clothing/suit/space/santa
+	head = /obj/item/clothing/head/santa
+	back = /obj/item/storage/backpack/santabag
+	mask = /obj/item/clothing/mask/breath
+	r_pocket = /obj/item/device/flashlight
+	gloves = /obj/item/clothing/gloves/color/red
+	belt = /obj/item/tank/internals/emergency_oxygen/double
+	id = /obj/item/card/id/gold
+
+/datum/outfit/santa/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(visualsOnly)
+		return
+	H.real_name = "Santa Claus"
```

Comment:
use fully_replace_character_name instead. This doesn't hit the dna & stored_dna name otherwise.